### PR TITLE
Add delete all orders

### PR DESCRIPTION
**Pull request by rodesousa**
add: DELETE /api/v3/openOrders

source: https://github.com/binance/binance-spot-api-docs/blob/master/rest-api.md#cancel-all-open-orders-on-a-symbol-trade

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,8 @@ end
 
 ```
 config :binance,
-  api_key: "xxx",
-  secret_key: "xxx",
+  api_key: System.get_env("BINANCE_API_KEY") || raise("BINANCE_API_KEY"),
```
[the remark this answers is not in the document]
Comment (rodesousa) [reply README.md]: Yep, done !

File: lib/binance.ex [hunk 2]
```diff
@@ -783,4 +783,44 @@ defmodule Binance do
...
+
+    arguments =
+      %{
+        symbol: symbol,
+        timestamp: timestamp
+      }
+      |> Map.merge(unless(is_nil(recv_window), do: %{recvWindow: recv_window}, else: %{}))
+
+    arguments
+    |> IO.inspect()
```
Comment (dvcrn) [inline lib/binance.ex:814]: Can you remove the IO.inspect here? I guess this is leftover from dev

Comment (rodesousa) [reply lib/binance.ex]: done

**Comment by dvcrn** [comment]
Apologies for the back and forth but I just now noticed that there are no tests to validate this. Do you think you can add a exvcr cassette for this? For example: https://github.com/dvcrn/binance.ex/blob/master/fixture/vcr_cassettes/cancel_order_by_struct_symbol_orderid_and_timestamp_success.json

Then in test: https://github.com/dvcrn/binance.ex/blob/master/test/binance_test.exs#L518-L545

If you're unsure how to use exvcr let me know and I can help with adding it as well! 👍

**Comment by dvcrn** [comment]
Superseded with https://github.com/dvcrn/binance.ex/pull/92 which added `delete_open_orders` - https://hexdocs.pm/binance/2.0.0-beta.1/Binance.Trade.html#delete_open_orders/2

Closing this for now

**Pull request closed without merging**